src/viam/sessions_client.py

```diff
@@ -37,6 +38,7 @@ class SessionsClient:
     channel: Channel
     client: RobotServiceStub
     _address: str
+    _robot_address: Optional[str]
```

Comment:
We could probably get away with not storing this, since only one of `_address` and `_robot_address` is ever necessary, but the logic got slightly more finicky and less readable, and the cost of storing the address here is very small, so it's probably not a big deal and not worth flagging here but it was annoying me so I'm articulating it!

Reply:
I would perhaps add a comment here differentiating between these 2. It took me a second to figure it out, so would be good to leave breadcrumbs for our future selves